src/ralph/backends/http/lrs.py

```diff
@@ -4,25 +4,47 @@
 from ralph.backends.http.async_lrs import AsyncLRSHTTP
 
 
+def _check_for_running_loop(func):
```

Comment:
Maybe this would be more explicit?

```suggestion
def _ensure_running_loop_uniqueness(func):
```